app/domain/etl/ga/views_and_navigation_service.rb

```diff
@@ -23,6 +23,8 @@ def client
   def append_data_labels(values)
     page_path, pviews, upviews, entrances, exits, bounces, page_time = *values
 
+    page_path = page_path.slice(1500) if page_path.length > 1500
```

Comment:
Could you please extract a constant with a meaningful name for the max path length?

Reply:
Yes sorry for the magic number - [fixed](https://github.com/alphagov/content-performance-manager/commit/cc0c35e8add2d9086b41d0b57b018c967951d20d)